cdap-ui/app/cdap/components/PipelineScheduler/Store/index.js

```diff
@@ -0,0 +1,210 @@
+
+      return {
+        ...state,
+        cron: cronArray.join(" ")
+      };
+    }
+    case ACTIONS.SET_STATE:
+      return {
+        ...state,
+        ...action.payload
+      };
+    case ACTIONS.SET_INTERVAL_OPTION:
+      return {
+        ...state,
+        intervalOption: action.payload.intervalOption
+      };
+    case ACTIONS.SET_MINUTE_INTERVAL:
+      return {
+        ...state,
+        minInterval: parseInt(action.payload.minInterval, 10)
+      };
+    case ACTIONS.SET_HOUR_INTERVAL:
+      return {
+        ...state,
+        hourInterval: parseInt(action.payload.hourInterval, 10)
+      };
+    case ACTIONS.SET_DAY_INTERVAL:
+      return {
+        ...state,
+        dayInterval: parseInt(action.payload.dayInterval, 10)
+      };
+    case ACTIONS.SET_DAYS_OF_WEEK_INTERVAL: {
+      let daysOfWeekInterval = [...state.daysOfWeekInterval];
+      let dayOption = action.payload.dayOption;
+      let dayOptionIndex = daysOfWeekInterval.indexOf(dayOption);
+      if (dayOptionIndex !== -1) {
+        daysOfWeekInterval.splice(dayOptionIndex, 1);
+      } else {
+        daysOfWeekInterval.push(dayOption);
+      }
+      daysOfWeekInterval.sort();
+      return {
+        ...state,
+        daysOfWeekInterval
+      };
+    }
+    case ACTIONS.SET_DATE_OF_MONTH_INTERVAL:
+      return {
+        ...state,
+        dateOfMonthInterval: parseInt(action.payload.dateOfMonthInterval, 10)
+      };
+    case ACTIONS.SET_MONTH_INTERVAL:
+      return {
+        ...state,
+        monthInterval: parseInt(action.payload.monthInterval, 10)
+      };
+    case ACTIONS.SET_STARTING_AT_MINUTE:
+      return {
+        ...state,
+        startingAtMinute: parseInt(action.payload.startingAtMinute, 10)
+      };
+    case ACTIONS.SET_STARTING_AT_HOUR:
+      return {
+        ...state,
+        startingAtHour: parseInt(action.payload.startingAtHour, 10)
+      };
+    case ACTIONS.SET_STARTING_AT_AM_PM:
+      return {
+        ...state,
+        startingAtAMPM: action.payload.startingAtAMPM
+      };
+    case ACTIONS.SET_MAX_CONCURRENT_RUNS:
+      return {
+        ...state,
+        maxConcurrentRuns: parseInt(action.payload.maxConcurrentRuns, 10)
+      };
+    case ACTIONS.SET_SCHEDULE_VIEW:
+      return {
+        ...state,
+        scheduleView: action.payload.scheduleView
+      };
+    case ACTIONS.RESET:
+      return DEFAULT_SCHEDULE_OPTIONS;
+  }
+};
+
+const composeEnhancers =
+typeof window === 'object' &&
+window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ?
+  window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
+    name: 'PipelineSchedulerStore'
+  }) : compose;
+
+const PipelineSchedulerStore = createStore(
+  schedule,
+  DEFAULT_SCHEDULE_OPTIONS,
+  composeEnhancers()
+);
+
+export default PipelineSchedulerStore;
+export {INTERVAL_OPTIONS, MINUTE_OPTIONS, HOUR_OPTIONS, HOUR_OPTIONS_CLOCK, DAY_OF_WEEK_OPTIONS, DATE_OF_MONTH_OPTIONS, MONTH_OPTIONS, AM_PM_OPTIONS, MAX_CONCURRENT_RUNS_OPTIONS, ACTIONS};
```

Comment:
Have new line separate the named exports. Easier to read. Just like the `ActionCreator`s we have